x/ccv/provider/client/cli/tx.go

```diff
@@ -215,17 +215,17 @@ Example:
 
 func NewCreateConsumerCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "create-consumer [chain-id] [metadata] [initialization-parameters] [power-shaping-parameters]",
+		Use:   "create-consumer [consumer-parameters]",
```

Comment:
How will the user differentiate between the json they need to provide for create-consumer and the one for update-consumer?

Reply:
right, we need to provide an example.
I can also imagine to have the common part's in a json file (e.g. metadata, init-, powershaping-parameters) and those which differ as mandatory/optional arguments (optional would be 'new_owner').
If you're ok I'll keep this change for later once we have a preference. 
In any case we'll end up with a 'one-file solution' I'd say.